### DISCUSSION TITLE
Change top bar title?

### DISCUSSION THREAD

**Discussion by ChaoticPumpkin** [thegeeklab/hugo-geekdoc#575] [Q&A] [answered]
I was looking through the documentation page on [Site Configuration](https://geekdocs.de/usage/configuration/#site-configuration), and I couldn't find anything about changing the title in the top bar, is there any way to do that in the config? 
![Screenshot from 2023-02-01 17-46-49](https://user-images.githubusercontent.com/59938482/216211748-2ad0e7eb-c7ca-4ade-bd4b-42cd1598cb30.png)
I could be totally missing something obvious, but any help would be appreciated!

**Comment by xoxys** [comment, marked as answer]
It's in the second line `title = "Geekdocs"`.

**Reply by ChaoticPumpkin** [reply]
Oh my god, how did I miss that? Thanks so much!

**Reply by xoxys** [reply]
No worries 🙂 Happy to help
